### PR TITLE
Improved UX on home cards

### PR DESCRIPTION
**Pull request by Molizanee**
- Added drop-shadow on hover, cursor-pointer and a description to "Open Database" card.

### CODE REVIEW

File: package-lock.json
Comment (xp4u1) [inline package-lock.json:1]: Please use yarn as package manager. There will be conflicts when we mix npm and yarn

File: cypress/screenshots/test.cy.ts/My First Test -- Visits the app root url (failed).png
Comment (xp4u1) [inline cypress/screenshots/test.cy.ts/My First Test -- Visits the app root url (failed).png:1]: Currently, there are no tests. There is no reason for this screenshot to be commited in this pull request.

File: package.json
Comment (xp4u1) [inline package.json:1]: Why did you downgrade this dependency?

File: src/pages/Home.tsx [hunk 1]
```diff
@@ -155,23 +155,23 @@ const Home: React.FC = () => {
...
-          <section className="cardContainer">
-            <div className="card" onClick={decideReflectionPrompt}>
-              <h1>{t("Reflection")}</h1>
-              <p>{t("What happened today?")}</p>
-            </div>
-            <div className="card" id="addMemoryTrigger">
-              <h1>{t("Memories")}</h1>
-              <p>{t("Capture thoughts and feelings")}</p>
-            </div>
+          <section className="cardContainer">
```
Comment (xp4u1) [inline src/pages/Home.tsx:158]: There are whitespace characters at the end of the line.

File: src/pages/Home.tsx
Comment (xp4u1) [inline src/pages/Home.tsx:1]: Please use a code formatter and set the tab width to 2 whitespaces. Personally, I use [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode). I should add a guideline for contributors.

File: src/pages/Home.scss [hunk 2]
```diff
@@ -119,7 +144,6 @@
 
 ion-alert.dayPrompt {
   button.alert-button:last-child {
-    // Override Ionic's "bold"
```
Comment (xp4u1) [inline src/pages/Home.scss:122]: Why did you remove this comment?

**Comment by xp4u1** [comment]
Hi Molizanee,

Thank you for your pull request and for taking the time to contribute to the project!

I noticed that there are more changes in this commit than the description suggests. In particular, since this commit is focused on UI changes, there shouldn't be any modifications to dependencies. However, I see that there are changes in the package.json file, including what appears to be a package downgrade. Could you please clarify why these changes are necessary?

Additionally, it would be great if you could use a code formatter like Prettier to ensure consistent code style throughout the project.

I also noticed that there are some untranslated strings in this commit. You don't necessarily need to add translations, but it would be good to keep in mind that all UI changes should ideally have corresponding translations to maintain consistency across different languages.

Regarding the commit message: `chore` isn't the most appropriate prefix for these changes since they primarily focus on improving the user experience. I suggest using `feat(ux)` as the prefix to better reflect the nature of the update.

Lastly, I haven't yet reviewed the UI/UX changes in detail and haven't been able to check whether everything looks and works as intended. I agree with your point that the button is not easy to see in the mobile version. However, I won't be able to make further changes or decisions on this today and probably not in the next few days, as I am still busy with other tasks.

Thank you again for your contribution! Looking forward to your updates.

**Review by xp4u1 — CHANGES_REQUESTED** [review]

**Comment by Molizanee** [comment]
Hi xp4u1,

I’ve made all the requested changes. Regarding the downgrade of the "pouchdb-authentication" package, it was necessary because I hadn't run the project with yarn in my first commit. I’ve now reverted it to version 1.1.3. Additionally, I added Prettier to format the code with 2-tab indentation as requested, and I restored the comment that I had previously removed.

This is my first contribution, so I really appreciate your time in explaining what I can improve and what I need to do. Thanks a lot!

**Review by xp4u1 — COMMENTED** [review]
Hi Molizanee,

I wanted to address a few points regarding this merge request:

1. **UI/UX Changes**: I cannot approve the current UI/UX changes. I’m still concerned about this approach on smaller screens of older devices. That was also the reason why the button is one of the large white buttons on larger screens. The implementation doesn’t align with my vision for the project and requires significant revisions. Thanks for your ideas and feedback, but in this case, I would prefer to reconsider the approach myself and develop a solution.

2. **Prettier Configuration**: The idea of creating a Prettier configuration is awesome, but it should be submitted as a separate pull request. Additionally, the `.prettierignore` file isn’t correct; please refer to my comments for details.

I appreciate your interest and enthusiasm for the project. However, with the pull request introducing +17,822/-698 changes, I can’t merge it as it stands. I don’t have the capacity to guide these changes, so I recommend focusing on smaller, more manageable contributions.

Thank you again for your efforts and understanding. I look forward to seeing your future contributions.

**Pull request closed without merging**